content/21.nutraceuticals.md

```diff
@@ -110,56 +112,60 @@ Therefore, the evidence is not present to draw conclusions about whether n-3 PUF
+In inflammatory situations such as an infection, zinc can regulate leukocyte immune responses and activate the nuclear factor kappa-light-chain-enhancer of activated B cells (NF-kB), thus altering cytokine production [@doi:10.4049/jimmunol.179.6.4180; @doi:10.1067/mlc.2001.118108].
 In particular, zinc supplementation can increase natural killer cell levels, which are important cells for host defense against viral infections [@doi:10.1126/science.1198687; @doi:10.1016/j.abb.2016.03.022].
+As a result of these immune-related functions, zinc is also under consideration for possible benefits against COVID-19.
 
 Adequate zinc intake has been associated with reduced incidence of infection [@doi:10.1093/ajcn/85.3.837] and antiviral immunity [@doi:10.1093/advances/nmz013].
 A randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found zinc deficiency to be associated with increased susceptibility to infection and that zinc deficiency could be prevented through supplementation [@doi:10.1093/ajcn/85.3.837].
 Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
-An observational study has shown that COVID-19 patients had significantly lower zinc levels in comparison to healthy controls, they also tended to have more complications (70.4% vs 30.0%, _p_ = 0.009), and potentially prolonged hospital stays (7.9 vs 5.7 days, _p_ = 0.048) [@doi:10.1016/j.ijid.2020.09.014].  
-In coronaviruses specifically, _in vitro_ evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV-1) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
-Currently, there are over twenty clinical trials registered with the intention to use zinc in a preventative or therapeutic manner.
+An observational study showed that COVID-19 patients had significantly lower zinc levels in comparison to healthy controls and that zinc-deficient COVID-19 patients (those with levels less than 80 μg/dl) tended to have more complications (70.4% vs 30.0%, _p_ = 0.009) and potentially prolonged hospital stays (7.9 vs 5.7 days, _p_ = 0.048) relative to patients who were not zinc deficient [@doi:10.1016/j.ijid.2020.09.014].
+In coronaviruses specifically, _in vitro_ evidence has demonstrated that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV-1) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
```

Comment:
Zn2+ should be superscript but I had no idea how to do it

Reply:
```suggestion
In coronaviruses specifically, _in vitro_ evidence has demonstrated that the combination of zinc (Zn^2+^) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV-1) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
```
This will do superscript -- for subscript, it would be `Zn~2+~` (I think! This always throws me!)